CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## HEAD
 
+* Add non-finite float values (+inf, inf, -inf, nan)
```

Comment:
Perhaps only list one of `+inf` and `inf` here – since this says "values" and those are the same value written with different syntaxes.

Reply:
Fair enough, will tighten in up.